src/styles/blocks/nav.scss

```
@@ -0,0 +1,48 @@
+.nav {
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+  }
+
+  &__link {
+    height: 60px;
+    white-space: nowrap;
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    color: var(--card-main);
+    font-weight: 500;
+    line-height: 14px;
+    text-transform: uppercase;
+    cursor: pointer;
+
+    &:hover {
+      color: var(--card-secondary);
+    }
+
+    &:active,
+    &--active { /* keeping these for possible tests */
+      position: relative;
+      color: var(--card-secondary);
+    }
+
+    &:active::after,
+    &--active::after { /* keeping these for possible tests */
+      content: "";
+      border-radius: 5px;
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      height: 4px;
+      background-color: var(--card-secondary);
+    }
+  }
+
+  &__item + &__item {
+    margin-left: 20px;
+  }
```

i see you're using this quite often
i'd even say i love it from logic perspective,
but using more explanatory syntax would just be more clear what you're targeting for
say
```suggestion
  .nav__item:not(:first-child) {
    margin-left: 20px;
  }
```

will also be easy to switch margins from -left to -rigth if needed:
```suggestion
  .nav__item:not(:last-child) {
    margin-right: 20px;
  }
```

but for this use case your approach does the job